util/cleanup.h

```diff
@@ -25,9 +27,15 @@ static inline void freep(void *p)
 }
 #define _cleanup_free_ __cleanup__(freep)
 
+static inline void free_huge(struct nvme_mem_huge *mh)
+{
+	nvme_free_huge(mh);
+}
```

Comment:
Could just use `nvme_free_huge()` instead of defining a new function

Reply:
Sure, fixed :)